scipy/stats/setup.py

```diff
@@ -21,35 +23,39 @@ def configuration(parent_package='',top_path=None):
 
     # add statlib module
     config.add_extension('statlib',
-        sources=['statlib.pyf'],
-        f2py_options=['--no-wrap-functions'],
-        libraries=['statlib'],
-        depends=statlib_src
-    )
+                         sources=['statlib.pyf'],
+                         f2py_options=['--no-wrap-functions'],
+                         libraries=['statlib'],
+                         depends=statlib_src)
 
     # add _stats module
     config.add_extension('_stats',
-        sources=['_stats.c'],
-    )
+                         sources=['_stats.c'], )
```

Comment:
While this is valid syntax, it is unusual to leave the trailing comma when it is immediately followed by `)`.  I think this would look better without the trailing comma, but I wouldn't block the PR based on that.  (This applies here and in several other changes below.)

(Note that it *is* fairly common to leave the redundant trailing comma as it was before this change, when the closing parenthesis is on a separate line.   That allows additional elements to be added to the list in new lines without having to add a comma to what was previously the last element.)

Reply:
My bad I forgot this, thanks!